### DISCUSSION TITLE
Conteúdos diversos

### DISCUSSION THREAD

**Discussion by SloppyMaster11** [YuriAlessandro/gerador-ficha-tormenta20#356] [Ideias]
Usando uma única thread pra compilar várias Edições da Caverna do Saber:

DB#145 - Regras de culinária, pratos especiais e um poder novo: 'Comilão'
DB#153 e DB#154 - Regras para personagens de idades variadas 
DB#161 - Regras de Aposentadoria para personagens
DB#166 - Regras novas para Poções e poderes: 'Cozinhar Receita Mágica' e 'Destilar Substrato'; E equipamento: 'Lançador Alquímico'
DB#168 - Aprimoramentos para itens gerais

**Comment by YuriAlessandro** [comment]
Isso cobre todos os conteúdos desde a 145? Ou são só alguns selecionados?

**Reply by SloppyMaster11** [reply]
Apenas alguns selecionados.
Meus amigos e eu passamos algum tempo checando edições antigas da DB, mas muitos conteúdos da CdS eram para o TRPG.
Selecionamos apenas os mais interessantes, no nosso ver, e que eram para o T20.
Ainda voltarei aqui e enviarei mais sugestões de outras edições.

**Reply by YuriAlessandro** [reply]
Show, só queria saber para poder colocar lá certinho quais já foram analisadas. Vou adicionar assim que possível.
